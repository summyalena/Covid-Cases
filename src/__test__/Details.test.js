import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Details from '../components/details/details';
import store from '../redux/configureStore';

it('testing-covidCase', () => {
  const component = renderer.create(
    <Provider store={store}>
      <Details />
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
