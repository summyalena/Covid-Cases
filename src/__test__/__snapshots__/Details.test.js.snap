// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing-covidCase 1`] = `
<div>
  <div />
  <div
    className="breakdown"
  >
    <h4>
      Country Breakdown
    </h4>
  </div>
</div>
`;
